src/themes/material/templates/body.cell.reference.edit.js

```diff
@@ -30,25 +30,26 @@ export default function ReferenceEdit($scope, qgrid) {
 		}
 	});
 
-	this.commit = ($cell, $event) => {
-		const model = this.gridModel;
-		const selectionItems = model.selection().items;
-		const entries = new SelectionService(model).lookup(selectionItems);
-
-		const cell = this.cell();
-		cell.value = model.selection().items;
-		cell.tag = {
-			entries: entries,
-			columns: model.data().columns
-		};
-		cell.commit.execute($cell, $event);
-
-		close();
-	};
-
-	this.cancel = ($cell, $event) => {
-		this.cell().cancel.execute($cell, $event);
-		close();
+	this.commands = {
+		commit: new Command({
+			shortcut: 'ctrl+s',
+			execute: ($cell, $event) => {
+				this.cell().value = this.gridModel.selection().items;
+				this.cell().tag = {
+					entries: this.gridModel.selection().entries,
+					schema: this.gridModel.data().columns
```

Comment:
Nice catch, thank you! 
Seems that auto merge hadn't seen code changes conflict here